README.md

```diff
@@ -105,10 +59,10 @@ Note, these policies are broader than required for the deployment of Crunchbase.
 
 
 ### Installation
-Check out [this repository](https://github.com/UtrechtUniversity/ia-webscraping), and make sure you checkout the 'main' branch. Open a terminal window and navigate to the `cdx-records` directory.
+Check out [this repository](https://github.com/UtrechtUniversity/ia-webscraping), and make sure you checkout the 'main' branch. Open a terminal window and navigate to the `code` directory.
 ```bash
-# Go to CDX records folder
-$ cd cdx-records
+# Go to code folder
+$ cd code
 ```
 (The index format used by the Internet Archive is called 'cdx', hence the name)
```

Comment:
Now that we've renamed the folder 'code', this line is a little misplaced. Maybe better just remove it.